pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/rebalance/TableRebalancer.java

```diff
@@ -98,8 +98,15 @@ public RebalanceResult rebalance(TableConfig tableConfig, Configuration rebalanc
     }
 
     // Check that table exists and is enabled
-    IdealState currentIdealState =
-        _helixDataAccessor.getProperty(_helixDataAccessor.keyBuilder().idealStates(tableNameWithType));
+    IdealState currentIdealState;
+    try {
+      currentIdealState =
+          _helixDataAccessor.getProperty(_helixDataAccessor.keyBuilder().idealStates(tableNameWithType));
+    } catch (Exception e) {
+      LOGGER.warn("Caught exception while fetching IdealState for table: {}", tableNameWithType, e);
```

Comment:
Since these exceptions are caused by operator action, we should log them as INFO, not WARN

Reply:
These exceptions are unusual system exception instead of invalid operator action (ZK exception instead of table does not exist). Also, real-time segment relocation is also done by TableRebalancer, so I would not treat it as a user facing tool. Same for other WARNING. For invalid operator action such as table does not exist/rebalance HLC table, no warning is logged.